### PR TITLE
PS-8554: Introduce PS compilation testing on arm64 with Cirrus CI

### PR DESCRIPTION
**Pull request by inikep**
1. Introduce `.cirrus.yml`.
2. Add two workers (Debug and RelWithDebInfo) to compile Percona Server with gcc on arm64.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    cd $WORKING_DIR/bin
+    NPROC=`nproc --all`
+    NTHREADS=$(( $NPROC > 16 ? 16 : $NPROC ))
+    if [[ "$BUILD_TYPE" == "RelWithDebInfo" ]]; then
+      MTR_TESTS="--suite=main"
+    else
+      MTR_TESTS="main.1st"
+    fi
+    echo Start testing Using $NTHREADS/$NPROC threads; MTR_TESTS=$MTR_TESTS
+    mysql-test/mysql-test-run.pl $MTR_TESTS --parallel=$NTHREADS --mysqld-env=LD_PRELOAD=/usr/lib/aarch64-linux-gnu/libeatmydata.so --force --max-test-fail=0 --retry-failure=0 --debug-server && echo Ignore mysql_test_run.pl errorcode
```
Comment (percona-ysorokin) [inline .cirrus.yml]: Did you mean `||` instead of `&&`?

File: .cirrus.yml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    lsblk
+    lsblk -f
+    df -Th
+    mkdir -p $WORKING_DIR
+  << : *SCRIPT_TEMPLATE
+
+task:
+  << : *FILTER_TEMPLATE
+  aws_credentials: ENCRYPTED[!92ac22d2430cf40dfcec42f739513a65c8b368c822cb397e95f799d41c0ba4498c3a1c337ab14a25cc47b2d4b53c46c5!]
+  only_if: $CIRRUS_REPO_FULL_NAME == "inikep/percona-server"
```
Comment (percona-ysorokin) [inline .cirrus.yml:124]: Is `inikep` intentional here?

File: .cirrus.yml [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    sudo mkdir $MOUNT_POINT
+    sudo mount /dev/nvme1n1 $MOUNT_POINT
+    sudo mkdir -p $WORKING_DIR
+    df -Th
+  << : *SCRIPT_TEMPLATE
+
+task:
+  << : *FILTER_TEMPLATE
+  aws_credentials: ENCRYPTED[!92ac22d2430cf40dfcec42f739513a65c8b368c822cb397e95f799d41c0ba4498c3a1c337ab14a25cc47b2d4b53c46c5!]
+  only_if: $CIRRUS_REPO_FULL_NAME == "DISABLED-inikep/percona-server"
```
Comment (percona-ysorokin) [inline .cirrus.yml:157]: Is `inikep` intentional here?

File: .cirrus.yml (hunk 2 above)
Comment (inikep) [reply .cirrus.yml:124]: Yes, I want only `arm_container` to be used by `percona/percona-server`.

File: .cirrus.yml (hunk 3 above)
Comment (inikep) [reply .cirrus.yml:157]: Yes, I want only `arm_container` to be used by `percona/percona-server`.

File: .cirrus.yml (hunk 1 above)
Comment (inikep) [reply .cirrus.yml]: Yes, you are right, but I will remove `echo Ignore mysql_test_run.pl errorcode` because it's not needed now

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**